examples/get-started/vite.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+
+// https://vitejs.dev/config/
```

Comment:
is this here for a quick access to reference?

Reply:
This was part of the vite codegen... probably just for reference